src/nvim/api/win_config.c

```diff
@@ -1023,6 +1023,18 @@ static void parse_border_style(Object style, WinConfig *fconfig, Error *err)
   }
 }
 
+static void generate_api_error(win_T *wp, const char *attribute, Error *err)
+{
+  char fmt[256];
+  if (wp->w_floating) {
+    snprintf(fmt, sizeof(fmt), "Missing relative field when reconfiguring floating window %d",
+             wp->handle);
+  } else {
+    snprintf(fmt, sizeof(fmt), "non-float cannot have '%s'", attribute);
```

Comment:
Can't you use `api_set_error()` here directly?

Reply:
The first time this was done. some line breaks are needed

Reply:
oh yup ..